mis_builder/tests/test_aep.py

```diff
@@ -214,15 +215,15 @@ def test_aep_convenience_methods(self):
             self.account_in.id: (0, 500),
         })
         end = AEP.get_balances_end(
-            self.company,
+            self.company_ids,
             time.strftime('%Y') + '-03-31',
             'posted')
         self.assertEquals(end, {
             self.account_ar.id: (900, 0),
             self.account_in.id: (0, 800),
         })
         unallocated = AEP.get_unallocated_pl(
-            self.company,
+            self.company_ids,
```

Comment:
So you the the AEP __init__ method shoud accept company and companies as first argument and I shoud add check whether theare are one or many companies passed?